src/module/actor/SR5Actor.ts

```diff
@@ -2255,11 +2255,31 @@ export class SR5Actor<SubType extends Actor.ConfiguredSubType = Actor.Configured
      * Delete References to this actor and all owned items in the Storage areas
      */
     async deleteStorageReferences(this: SR5Actor) {
+        // If the actor being deleted has a lot of items, this can take some time
```

Comment:
Parts of this already use flows, the overall implementation should as well. Likely the `MatrixNetworkFlow`.

Again, just to keep the document classes small.

I'm currently mainly promoting my view point around this topic of Document entrypoint and Flow as a kind of facade pattern and flow / rule methods as imlementations. If you and / or numbers disagree, please let me know. :)

Reply:
Good call! I'll clean that up. And I like the design as well, although the circular code of Flow => Actor => Flow can feel weird sometimes.